src/toil/test/src/importExportFileTest.py

```diff
@@ -144,5 +144,5 @@ def run(self, file_store):
             with file_store.readGlobalFileStream(self.msg_portion_file_id, encoding='utf-8') as readable:
                 # combine readable.read() (the original message 1) with message 2
                 # this will be the final output of the workflow
-                writable.write(f'{readable.read()}{self.message_portion_2}'.encode('utf-8'))
+                writable.write(f'{readable.read()}{self.message_portion_2}'.encode())
```

Comment:
How come it wants to remove explicit `'utf-8'` encodings when we encode stuff?

Reply:
That's the default encoding from a certain version of Python onward: https://github.com/asottile/pyupgrade/pull/485